### DISCUSSION TITLE
any way to change pen stroke's width?

### DISCUSSION THREAD

**Discussion by zillionare** [slidevjs/slidev#1049] [Q&A] [answered]
slidev is great and I love it!

Just need a way to change pen stroke's width during text annotation, even statically configured is ok.
I know it uses drauu but sadly I'm not familiar with this tool

**Comment by zillionare** [comment]
I've tried to change localStorage('slidev-drawing-brush'), doesn't seem to work

**Reply by zillionare** [reply]
understand now. The pen's stroke width can be set by brush.size (modify source code directly). My issue is caused by hardware, I've used a android mobile phone, it's poor performed on touch pen

**Comment by kermanx** [comment, marked as answer]
This function has been added via #1319
